Tasks/Shurko/Task3/Task3.py

```diff
@@ -0,0 +1,59 @@
+while True:
+    s_long = input ("Enter string length from 35 to 50: ")
+    if not s_long.isdigit():
+        print ('Incorrect input, must be digit')
+        continue
+    s_long = int(s_long)
+    if s_long < 35 or s_long > 50:
+        print ('Incorrect input, its out of range')
+        continue
+    break
+
+form_srt = ""
+f_name="text_formated.txt"
+
+f = open (f_name, 'w')
+f.close()
```

Comment:
I assume you need it to then append something? Append on itself can create file as well

And if you want to do it this way, better use **with open(....) as f: pass**